src/requirements.txt

```diff
@@ -0,0 +1,33 @@
+django==3.2.13
```

Comment:
Let's make this `3.2.15` which I believe is LTS

Reply:
done.

Reply:
I'm a little worried by having two requirements files...